src/site/xdoc/server/feature-mailetcontainer.xml

```diff
@@ -47,16 +47,18 @@
       to a local repository, or handling remote delivery.  Both the Matcher and Mailet APIs are 
       public, allowing James users to write their own custom matchers and mailets.  James 
       comes with a large set of pre-built matchers and mailets.</p>
-      
-      <p>Matchers and mailets are used in pairs.  At each stage in processing a message is checked 
+
+      <p>Matchers and mailets are used in pairs.  At each stage in processing a message is checked
       against a matcher.  The matcher will attempt to match the mail message.  The match is not simply
-      a yes or no issue.  Instead, the match method returns a collection of matched recipients.  If the 
+      a yes or no issue.  Instead, the match method returns a collection of matched recipients.  If the
       this collection of matched recipients is empty, the mailet is not invoked.  If the collection of
-      matched recipients is the entire set of original recipients, the mail is then processed by the 
+      matched recipients is the entire set of original recipients, the mail is then processed by the
       associated mailet.  Finally, if the matcher only matches a proper subset of the original recipients,
       the original mail is duplicated.  The recipients for one message are set to the matched recipients,
       and that message is processed by the mailet.  The recipients for the other mail are set to the
       non-matching recipients, and that message is not processed by the mailet.</p>
+
+      <p>Note that you can specify per mailet and per matcher detailed error handling logic.</p>
```

Comment:
no link?

Reply:
That differs only from an ancor from a link specified above. Anyway, added.